src/stores/repository-core/index.ts

```diff
@@ -157,6 +167,123 @@ const createRepoCoreSlice: StateCreator<CombinedSlices, [['zustand/immer', never
         })
       }
     },
+    inviteContributor: async (address: string) => {
```

Comment:
Maybe we should throw errors from `addContributor`, `inviteContributor` and other repo actions because its using `withAsync` and when these actions are used in components, no error is thrown when error occurs and we cannot catch them.

Reply:
handled